Vladimir_Mustafa/HW3/03.1 PRACTICAL TASKS.py

```diff
@@ -0,0 +1,50 @@
+# Task 1
+zen_line = input('Enter line:\n').lower()
+
+count_better = 0
+count_never = 0
+count_is = 0
+
```

Comment:
good job 
but more optimal will be to use method count() like this
print("In 'Zen of Python' the word \"better\" repeat", zen_lin.count("better"), 'times')
print("In 'Zen of Python' the word \"never\" repeat", zen_lin.count("never"), 'times')
print("In 'Zen of Python' the word \"is\" repeat", zen_lin.count("is"), 'times')